### ISSUE TITLE
Compression of state column in jv_snapshot

### ISSUE THREAD

**Issue by oldirtybasti** [javers/javers#947]
Thank you very much for this great piece of software. I am using it with an aurora aws serverless database. It would be great to have the ability to compress the state column in jv_snapshot to save space and get better I/O with less traffic to keep costs down.

**Comment by bartoszwalacik** [comment]
Good idea, we are open for contributions. Nice to hear that Javers works on AWS Aurora

**Comment by oldirtybasti** [comment]
on a sidenote: I have to load the whole data history at the beginning of a test. I tried to load every item by InstanceID massively concurrently or fetch all SnapShots by Class in one go. Whats faster? findSnapshots or findShadows? Both would work for me
